source/includes/_addax.md

```diff
@@ -0,0 +1,186 @@
+# Zinc Managed Accounts
+
+## Overview
+
+Zinc's Managed Accounts allows customers to place orders without worrying about
+creating and managing Amazon accounts. Instead, API users can fund a balance
+with Zinc using PayPal or wire transfer. A network of Pro-buyers will then place
+orders on your behalf.
+
+Managed accounts is currently available for Amazon and Amazon UK.
+
+## Using Managed Accounts
+
+Zinc Managed Accounts is currently available by invite only. Please contact
+sales@zinc.io and mention Managed Accounts. You can also contact this address if
+you need more information to determine if Managed Accounts are right for you.
+
+## Funding your account
```

Comment:
We should at some point differentiate between funding the account balance and the fee we charge for each order, which can be separate.